From 93fb18f09ef1c6f78690abad4c126e82e5ca6eda Mon Sep 17 00:00:00 2001
From: Hongcheng Xie <hongcheng.xie@intel.com>
Date: Thu, 3 Sep 2020 14:06:13 +0800
Subject: [PATCH] Disable bpf for AiC environment.

bpf for network metrics is not suitable for AiC enviroment for multiple
instances.

Change-Id: I13e2e860fc9c93b9d5eea4189039b5631db22416
Signed-off-by: Hongcheng Xie <hongcheng.xie@intel.com>
---
 bpfloader/bpfloader.rc                | 1 -
 libbpf_android/include/bpf/BpfUtils.h | 2 +-
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/bpfloader/bpfloader.rc b/bpfloader/bpfloader.rc
index 04d9b81..3eb4e2d 100644
--- a/bpfloader/bpfloader.rc
+++ b/bpfloader/bpfloader.rc
@@ -57,6 +57,5 @@ service bpfloader /system/bin/bpfloader
     #
     rlimit memlock 1073741824 1073741824
     oneshot
-    reboot_on_failure reboot,bpfloader-failed
     # we're not really updatable, but want to be able to load bpf programs shipped in apexes
     updatable
diff --git a/libbpf_android/include/bpf/BpfUtils.h b/libbpf_android/include/bpf/BpfUtils.h
index 6814f94..028003a 100644
--- a/libbpf_android/include/bpf/BpfUtils.h
+++ b/libbpf_android/include/bpf/BpfUtils.h
@@ -169,7 +169,7 @@ std::string BpfLevelToString(BpfLevel BpfLevel);
 BpfLevel getBpfSupportLevel();
 
 inline bool isBpfSupported() {
-    return getBpfSupportLevel() != BpfLevel::NONE;
+    return false;
 }
 
 #define SKIP_IF_BPF_NOT_SUPPORTED                                                    \
-- 
2.25.1

